frontend/cypress/integration/common/service_details.ts

```diff
@@ -13,85 +13,86 @@ Given('a service in the cluster with a healthy amount of traffic', function () {
     this.targetService = SERVICE;
 });
 
-Then('user sees a list with content {string}', (tab: string) => {
+Then('sd::user sees a list with content {string}', (tab: string) => {
```

Comment:
We talked some offline about this. Newer versions of cypress cucumber detect and throw an error for multiple definitions of the same step definition but until we upgrade namespacing will work for keeping them separate.

Reply:
This poor-man's namespacing caught several name-clash issues.  The upgrade is still good, although taking this sort of explicit approach makes it very clear that the steps in the associated .ts file are expected to be used.

Reply:
I'll merge assuming the relevant CI passes.